vos/vos/commands/vcat.py

```diff
@@ -1,53 +1,61 @@
-"""cat files from vospace to stdout"""
-
+"""cat VOSpace DataNode to stdout"""
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from optparse import OptionParser
 import sys
-import os
 import logging
-from vos.commonparser import CommonParser
-from vos import vos, version
+from ..vos import Client
+from ..commonparser import CommonParser, set_logging_level_from_args, exit_on_exception
 
 
-def _cat(vospace_uri, cert_filename=None):
-    """Cat out the given uri."""
+def _cat(uri, cert_filename=None):
+    """Cat out the given uri stored in VOSpace.
+    
+    :param uri: the VOSpace URI that will be piped to stdout.
+    :type uri: basestring
+    :param cert_filename: filename of the PEM certificate used to gain access.
+    :type cert_filename: basestring
+    """
 
     fh = None
     try:
-        if vospace_uri[0:4] == "vos:":
-            fh = vos.Client(vospace_certfile=cert_filename).open(vospace_uri, view='data')
+        if uri[0:4] == "vos:":
+            fh = Client(vospace_certfile=cert_filename).open(uri, view='data')
         else:
-            fh = open(vospace_uri, 'r')
+            fh = open(uri, str("r"))
         sys.stdout.write(fh.read())
     finally:
         if fh:
             fh.close()
 
+DESCRIPTION = """Write the content of source (eg. vos:Node/filename) to stdout.
+
+Accepts cutout syntax for FITS files; see vcp --help for syntax details"""
 
-def vcat():
-    usage = "%prog [options] vos:VOSpace/node_name"
-    description = "Writes the content of vos:VOSpace/node_name to stdout."
 
-    parser = CommonParser(usage, description=description)
-    parser.add_option("-q", help="run quietly, exit on error without message", action="store_true")
+def vcat():
 
-    (opt, args) = parser.parse_args()
-    parser.process_informational_options()
+    parser = CommonParser(description=description)
```

Comment:
description is not defined and it's currently failing.

Isn't it simpler to make the description the doctring of the vcat function and pass it to CommonParser as CommonParser(description=vcat.__doc__)? (double underscore around doc)